keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -71,7 +74,9 @@ data class RedBlack(
     )
 }
 
-object Highlander : ClusterDeployStrategy() {
+data class Highlander(
+  override val considerOnlyAmazonHealth: Boolean = false
```

Comment:
My suggestion is `considerOnlyInstanceHealth`, to make it cloud-provider-agnostic.